src/Command/AllCommand.php

```diff
@@ -111,7 +111,8 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             }
 
             foreach ($tables as $table) {
-                $subArgs = new Arguments([$table], $options, ['name']);
+                $parser = $command->getOptionParser();
+                $subArgs = new Arguments([$table], $options, $parser->argumentNames());
```

Comment:
This change is necessary to the avoid the `..argument not defined..` exception added in Cake 5.1. We need to ensure the command is executed with all it's defined args and not just `name`.